doc/reference/reference_lua/box_schema/user_disable.rst

```diff
@@ -0,0 +1,29 @@
+.. _box_schema-user_disable:
+
+===============================================================================
+box.schema.user.disable()
+===============================================================================
+
+.. module:: box.schema
+
+.. function:: box.schema.user.disable(username)
+
+    Revokes ``usage`` and ``session`` permissions from the subject user. Equivalent to the following call:
+
+    ..  code-block:: lua
+        
+        box.schema.user.revoke('{username}','usage,session','universe',nil,{if_not_exists=true})            
+    
+    .. NOTE::
+
+       * ``session`` - (cannot be granted to a role) if is not granted, ``IPROTO_AUTH`` always fails connection to the user, so does ``box.session.su()``
+
+       * ``usage`` - (cannot be granted to a role) lets user use their privileges on database objects (e.g. read, write and alter space)
```

Comment:
Nit: https://www.tarantool.io/en/doc/latest/contributing/docs/style/#don-t-use-i-e-and-e-g